tests/query_runner/test_basesql_queryrunner.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+from redash.query_runner import BaseSQLQueryRunner, BaseQueryRunner
+
+
+class TestBaseSQLQueryRunner(unittest.TestCase):
+
+    def setUp(self):
+        self.query_runner = BaseSQLQueryRunner(BaseQueryRunner)
```

Comment:
```suggestion
        self.query_runner = BaseSQLQueryRunner({})
```

(first parameter is configuration)